### PR TITLE
#7Task. valery.aleshka

### PR DESCRIPTION
**Pull request by Valeryaleshka**
https://travis-ci.org/github/Valeryaleshka/js-assignments/builds/715782528

### CODE REVIEW

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -33,7 +33,217 @@
  *
  */
 function* get99BottlesOfBeer() {
-    throw new Error('Not implemented');
+
+    let song99botles = [
```
Comment (KostyaKov) [inline task/07-yield-tasks.js]: This type of task requires an implementation without the copy-paste hardcode of repeated lines

File: task/07-yield-tasks.js [hunk 2]
```diff
@@ -47,7 +257,22 @@ function* get99BottlesOfBeer() {
...
+    let a = 1, b = 0, temp;
+
+    while (num >= 0){
+        yield b;
+        temp = a;
+        a = a + b;
+        b = temp;
+        num--;
+    }
+
```
Comment (KostyaKov) [inline task/07-yield-tasks.js]: Extra blank lines should be removed

File: task/07-yield-tasks.js [hunk 3]
```diff
@@ -33,7 +33,217 @@
...
+        'Take one down and pass it around, no more bottles of beer on the wall.',
+        'No more bottles of beer on the wall, no more bottles of beer.',
+        'Go to the store and buy some more, 99 bottles of beer on the wall.'
+    ];
+    
+    for(let i = 0; i < song99botles.length; i++){
+        yield song99botles[i];
+    }           
+
+    return 0;
```
Comment (KostyaKov) [inline task/07-yield-tasks.js]: "return" in this "function * generator" is redundant

File: task/07-yield-tasks.js [hunk 4]
```diff
@@ -47,7 +257,22 @@ function* get99BottlesOfBeer() {
...
+    while (num >= 0){
+        yield b;
+        temp = a;
+        a = a + b;
+        b = temp;
+        num--;
+    }
+  
+    
+    return a;
```
Comment (KostyaKov) [inline task/07-yield-tasks.js]: function * generator suggests using "yield"

**Review by KostyaKov — COMMENTED** [review]
I'll mark the PR as "Draft", please click "ready for review" when it will be finished. Thank you!

**Comment by Valeryaleshka** [comment]
I've fixed everything.
New Travis-ci build.
https://travis-ci.org/github/Valeryaleshka/js-assignments/builds/721852562

**Review by KostyaKov — COMMENTED** [review]
Closing the PR as accepted! Thanks!

**Pull request closed without merging**